lib/rules/require-early-return.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(options) {
+        assert(
+            options === 'noElse',
```

Comment:
How about we would use `true` value? We can introduce `strict` later. To be honest, i don't see anything criminal in the second case.